src/script.h

```diff
@@ -160,6 +160,14 @@ class Script : public RecipeImpl {
     engine_info_.required_features.push_back(feature);
   }
 
+  /// Checks is |feature| is in required features
+  bool IsRequiredFeature(const std::string& feature) const {
+      return std::find(
+          engine_info_.required_features.cbegin(),
```

Comment:
Why not just `begin` and `end`? Why `cbegin` and `cend`?

Reply:
changed to `begin`/`end`